windows/devices/uwb/UwbSession.cxx

```diff
@@ -32,6 +174,57 @@ UwbSession::ConfigureImpl(const uwb::protocol::fira::UwbSessionData& uwbSessionD
     }
 
     m_sessionId = uwbSessionData.sessionId;
+
+    // Populate the PUWB_SET_APP_CONFIG_PARAMS
+    auto sessionUwbMap = uwbSessionData.uwbConfiguration.GetValueMap();
+
+    std::vector<std::shared_ptr<uint8_t>> datavector(ddi_service_map.size());
```

Comment:
Why `std::shared_ptr` instead of `std::unique_ptr`?

Reply:
No particular reason. I read up on the differences and will change the pointers accordingly.